src/components/Board/Board.tsx

```diff
@@ -0,0 +1,119 @@
+import clsx from "clsx";
+import {observer} from "mobx-react-lite";
+
+import styles from "./Board.module.scss";
+import Piece from "../Piece/Piece";
+import React, {useMemo} from "react";
+import {CellProps} from "../../interfaces/interfaces";
+import ThemeStore from "../../store/theme/ThemeStore";
+import {Theme} from "../../enums/Theme";
+import {
+    CELL_SIZE_DESKTOP_PX,
+    CELL_SIZE_DESKTOP_XL_PX,
+    CELL_SIZE_LAPTOP_PX,
+    CELL_SIZE_MOBILE_PX,
+    CELL_SIZE_PX,
+    CELL_SIZE_TABLET_PX
+} from "../../utils/consts/board";
+import useWindowSize from "../../utils/hooks/useWindowSize";
+import {SCREEN_SIZE} from "../../utils/consts/consts";
+
+interface BoardProps {
+    board: CellProps[];
+    currentPiece: CellProps | null;
+    onCellClick: (cell: CellProps) => void;
+    themeStore: ThemeStore;
+}
+
+const Board: React.FC<BoardProps> = observer((props): JSX.Element => {
+    const {
+        board,
+        currentPiece,
+        onCellClick,
+        themeStore,
+    } = props;
+
+    const { width: screenWidth } = useWindowSize();
+
+    const cells: JSX.Element[] = [];
+
+    let colPosition = 0;
+    let rowPosition = 0;
+
+    const cellSize = useMemo(() => {
+        if (screenWidth <= SCREEN_SIZE.mobile) {
+            return CELL_SIZE_MOBILE_PX;
```

Comment:
CSS can do this job for you

Reply:
If you'd use percents as we discussed, cell's absolute position can be set in percent as well

Reply:
https://codesandbox.io/s/react-fiddle-forked-ri2zty?file=/src/App.js:347-348

Reply:
Removed this variable